test/tools/router_check/test/config/Redirect.yaml

```diff
@@ -31,6 +31,10 @@ virtual_hosts:
   domains:
   - redirect.lyft.com
   routes:
+    - match:
+        prefix: /blah
+      route:
+        cluster: www2
```

Comment:
@LisaLudique, What is the purpose of this entry?

Reply:
/wait

Reply:
It's to (more clearly) demonstrate the correct path_redirect value now that this bug is fixed.